contracts/staking/tests/ValidatorManagerTests.t.sol

```diff
@@ -284,6 +278,18 @@ abstract contract ValidatorManagerTest is Test {
         _initializeEndValidation(validationID);
     }
 
+    function _testCompleteEndValidation(bytes32 validationID) internal virtual {
```

Comment:
I don't think we've had a practice of internal test methods like this, but open to the idea, and good to move forward with it imo to get merged